src/GitVersionCore/IncrementStrategyFinder.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace GitVersion
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+    using VersionCalculation.BaseVersionCalculators;
+    using LibGit2Sharp;
+
+    public enum CommitMessageVersionMode
```

Comment:
There does not appear to be a way to do it in YamlDotNet. `YamlMember` cannot be applied to enum members, and we are already using the hyphenate convention.